### DISCUSSION TITLE
Node v35 is buggy

### DISCUSSION THREAD

**Discussion by yasiryagi** [subquery/subql#1028] [Frontier TestNet - Bug Reporting]
I got indexer unhealth on the Polkadot project 

2022-05-13T09:12:40.240Z <health> ERROR Indexer is not healthy 
2022-05-13T09:12:50.253Z <health> ERROR Indexer is not healthy 
2022-05-13T09:12:55.149Z <benchmark> INFO NaN bps, target: #10284916, current: #10273228, estimate time: NaN days NaN hours NaN mins

811718be2ce2   onfinality/subql-query:v0.16.0          "/sbin/tini -- /usr/…"   8 days ago     Up 8 days               0.0.0.0:3004->3004/tcp, :::3004->3004/tcp   query_qmyr8xqgaxucxmp
9fa1fb1e7b76   onfinality/subql-node:v0.35.0           "/sbin/tini -- /usr/…"   8 days ago     Up 2 days (healthy)                                                 node_qmyr8xqgaxucxmp
70f54ae52a20   onfinality/subql-query:v0.15.0          "/sbin/tini -- /usr/…"   9 days ago     Up 9 days               0.0.0.0:3003->3003/tcp, :::3003->3003/tcp   query_qmszpq9f4u1gerv
f5ac4653fc92   onfinality/subql-node:v0.34.0           "/sbin/tini -- /usr/…"   9 days ago     Up 2 hours (healthy)                                                node_qmszpq9f4u1gerv

Polkadot was running node v35 and query v16. I updated to v35.2 and everything works now.

This is just to let you know that v35 is buggy and might want to remove it from the list.

**Comment by mzxyz** [comment]
Thanks for reporting this, our sdk will remove this buggy version from both npm and docker hub.
